src/main/kotlin/com/team4099/robot2022/subsystems/Wheel.kt

```diff
@@ -182,7 +180,7 @@ class Wheel(
     acceleration: LinearAcceleration = 0.0.meters.perSecond.perSecond
   ) {
     if (speed == 0.feet.perSecond) {
-      driveSpark.set(0.0)
+      driveFalcon.set(ControlMode.PercentOutput, 0.0)
     }
     var directionDifference =
         (direction - directionSensor.position).inRadians.IEEErem(2 * Math.PI).radians
```

Comment:
after switching to running the control loop on the rio, instead of using `directionSensor`, i would just use `directionAbsolute`

Reply:
skip this if we're not doing the control loop on the rio for now